### ISSUE TITLE
Spatialize information with AArcade

### ISSUE THREAD

**Issue by madjin** [M3-org/research#6]
I'm putting together a how-to guide on making nodes in AArcade from M3 research, here's the link: https://hackmd.io/08peA6QBSBWPImOtoLQSdA?both

**TL;DR**

1. Open `hub_construct` inside [Anarchy Arcade](https://store.steampowered.com/app/266430/)

![](https://i.imgur.com/0yD7amA.jpg)

2. Spawn stuff inside your box and pull the lever

![](https://i.imgur.com/3BIFJHO.jpg)

After we pack a few of these nodes with content to visualize a project we can spawn them in any other map, useful for video content creation and building a great showfloor for all of our projects.

Other ideas:

- Create a short tutorial video with voiceover to go from start to finish
- Collect weekly news into a running document, spatialize later into memory palace
- Visualize Github repos with nodes (like awesome-metaverse or individual projects)
